ignite/cmd/network_chain.go

```diff
@@ -9,7 +9,28 @@ import (
 func NewNetworkChain() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "chain",
-		Short: "Build networks",
+		Short: "Commands to launch chains",
+		Long: `The "chain" namespace features the most commonly used commands for launching
+blockchains with Ignite.
+
+As a coordinator you "publish" your blockchain to Ignite. When enough validators
+are approved for the genesis and no changes are excepted to be made to the
+genesis, a coordinator announces that the chain is ready for launch with the
+"launch" command. The chain launch can be cancelled with the "revert-launch"
+command.
+
+As a validator you "init" your node and apply to become a validator for a
+blockchain with the "join" command. After the launch of the chain is announced,
+validators can download the finalized genesis and a list of peers with the
```

Comment:
```suggestion
validators can generate the finalized genesis and download the list of peers with the
```
Generate is more appropriate for the genesis since the genesis is not directly stored in the Ignite chain